docs/candy-kata/slides.md

```diff
@@ -0,0 +1,125 @@
+<!--
+  ~ Copyright (c) 2018 Goldman Sachs and others.
```

Comment:
Update this to say "2021 Goldman Sachs and others."

Reply:
Updated